controllers/idler/idler_controller.go

```diff
@@ -34,6 +34,11 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 )
 
+const (
+	RestartThreshold     = 50
+	RequeueTimeThreshold = 300
```

Comment:
```suggestion
	RequeueTimeThresholdSeconds = 300
```

Reply:
done in [a9b2750](https://github.com/codeready-toolchain/member-operator/pull/630/commits/a9b27504aaa54ce3fd7aa66dad69abca3e0bf8f9)